lib/utils/testem-events.js

```diff
@@ -59,12 +59,12 @@ class TestemEvents {
    * @param {Array<string>} modules
    * @param {boolean} loadBalance
    * @param {boolean} replayExecution
-   * @param {string} appMode
+   * @param {string} executionOnly
    */
-  setModuleQueue(browserId, modules, loadBalance, replayExecution, appMode) {
+  setModuleQueue(browserId, modules, loadBalance, replayExecution, executionOnly) {
     const replayExecutionMap = this.stateManager.getReplayExecutionMap();
 
-    if (appMode === 'ci' && replayExecution) {
+    if (!executionOnly && replayExecution) {
```

Comment:
is `executionOnly` even needed here? Because with it, setBrowserModuleQueue won't be set. Is that expected?